GoMeet/src/main/java/com/google/sps/data/Location.java

```diff
@@ -8,15 +8,15 @@
   private final double lat;
   private final double lng;
   private final String note;
-  private int voteCount;
+  private long voteCount;
```

Comment:
Why is this a long? I'm assuming that you'd never maximum integer value.

Reply:
It was long because datastore stores the integer values as longs and getProperty("voteCount") would return a long. But, now that I think about it, I think it makes more sense to modify the data received from Datastore to suit my code than to modify my code to suit Datastore haha. I've changed it to int.

Reply:
Great, thanks for explaining. That sounds like a reasonable change to me.